index.bs

```diff
@@ -1385,38 +1383,45 @@ Build a composed graph up to a given output operand into a computational graph a
+
+            Issue(566): If {{MLGraphBuilder/constant(descriptor, bufferView)|constants'}} {{ArrayBuffer}}s are not [=transferred=], make copies for [=MLGraphBuilder/graph=]'s [=computational graph/constants=] here.
+
+    1. [=map/For each=] |name| → |operand| of |outputs|:
+        1. Set |graph|.{{MLGraph/[[outputDescriptors]]}}[|name|] to |operand|.{{MLOperand/[[descriptor]]}}.
+    1. Let |promise| be [=a new promise=].
     1. Run the following steps [=in parallel=]:
-        1. If |outputs| is empty, then [=queue an ML task=] with |global| to [=reject=] |promise| with a {{TypeError}}, and abort these steps.
-        1. [=map/For each=] |name| → |operand| of |outputs|:
-            1. If |name| is empty, then [=queue an ML task=] with |global| to [=reject=] |promise| with a {{TypeError}}, and abort these steps.
-        1. If any of the following sub-steps fail, then [=queue an ML task=] with |global| to [=reject=] |promise| with an "{{OperationError}}" {{DOMException}}, and abort these steps.
-            1. Let |graph| be a new {{MLGraph}} with |realm|:
-                1. Set |graph|.{{MLGraph/[[context]]}} to [=this=].{{MLGraphBuilder/[[context]]}}.
-            1. Make a request to the underlying platform to:
-                1. Connect |graph| to a new [=implementation-defined=] graph implementation |graphImpl| given |graph|.
-                1. Set |graph|.{{MLGraph/[[implementation]]}} to |graphImpl|.
-            1. Make a request to the underlying platform to initialize the graph:
-                1. [=map/For each=] |name| → |operand| of |outputs|:
-                    1. If [=MLOperand/validating MLOperand=] given |operand| and [=this=] returns false, then [=queue an ML task=] with |global| to [=reject=] |promise| with a {{TypeError}}, and abort these steps.
-                    1. If |operand| was created as an input by the underlying platform:
-                        1. If |operand|.{{MLOperand/[[name]]}} is not unique for |graphImpl|, then [=queue an ML task=] with |global| to [=reject=] |promise| with a {{TypeError}}, and abort these steps.
-                        1. Add |operand|.{{MLOperand/[[descriptor]]}} to |graph|.{{MLGraph/[[inputDescriptors]]}}[|operand|.{{MLOperand/[[name]]}}].
-                    1. If |operand| was created as a constant by the underlying platform:
-                        1. Implementations MAY preprocess and optimize the tensor data of |operand| for the underlying platform.
-                    1. Register a [=platform operand=] representing |operand| in |graphImpl| as graph output.
-                    1. Register a [=platform operator=] representing |operand|.{{MLOperand/[[operator]]}} to |graphImpl|.
-
-                Issue(552): Decide how to specify graph initialization.
+        1. Let |graphImpl| be the result of converting [=this=]'s [=MLGraphBuilder/graph=] into an [=implementation-defined=] format which can be interpreted by the underlying platform.
```

Comment:
I meant 
```suggestion
        1. Let |graphImpl| be the result of converting [=this=]'s [=MLGraphBuilder/graph=] with |operands|, |operators|, |inputs| and |outputs| into an [=implementation-defined=] format which can be interpreted by the underlying platform.
```

Reply:
Understood. When I was chatting with @reillyeon it seemed like he didn't think this level of detail was necessary since we're already passing the whole graph through and, unlike validating there are no duplicate input names, there's no script-visible behavior specific provided by more detail. I'd like him to weigh in when he's available again in a few days.

Reply:
If passing these parameters clarifies that the implementation is only supposed to consider a sub-graph of the full graph held by the builder that makes sense to me.

Reply:
Thanks @reillyeon !

5980228c513f4cea8a6d9c7ceba7667af6465c04 adds an `operands` and `operator` set. Note that the variable `outputs` is already used for the passed name &rarr; operands map, so I used the phrase `outputs's values` instead.

Reply:
looks good to me, thanks!